.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  push:
+    # Limit this action to tags only
+    tags:
+      - 'v[0-9]+\.[0-9]+\.[0-9]+'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
I think we should hardcode the version, so we know if it breaks when updating

```suggestion
    runs-on: ubuntu-20.04
```

Reply:
Nice catch thanks. Also changed the job name from` build` to `publish`